src/OpenDebugAD7/packages.config

```diff
@@ -4,4 +4,7 @@
   <package id="Microsoft.VisualStudio.Debugger.Interop.Portable" version="1.0.1" targetFramework="net45" />
   <package id="MicroBuild.Core" version="0.2.0" targetFramework="net45" />
   <package id="Mono.Unofficial.pdb2mdb" version="4.2.3.4" />
+
+  <!-- Update the version number VsCodeDebugProtocol HintPath in OpenDebugAD7\OpenDebugAD7.csproj -->
```

Comment:
><!-- Update the version number VsCodeDebugProtocol HintPath in OpenDebugAD7\OpenDebugAD7.csproj --> [](start = 2, length = 99)

I would suggest makes a .targets file instead, since .targets files are easier to edit.

Reply:
Theres only one for now and we aren't planning on updating it that often. Leaving it as is for now. 

---
In reply to: [206675075](https://github.com/Microsoft/MIEngine/pull/750#discussion_r206675075) [](ancestors = 206675075)